force use of -O1 when compiling ceos.c to avoid that Intel C compiler goes into infinite loop on Intel Skylake systems
see https://github.com/easybuilders/easybuild-easyconfigs/issues/5843
author: B. Hajgato (Free Univeristy Brussels - VUB)
--- gdal-2.2.2/frmts/ceos2/GNUmakefile.org	2017-09-15 12:37:53.000000000 +0200
+++ gdal-2.2.2/frmts/ceos2/GNUmakefile	2018-02-19 21:42:14.864273222 +0100
@@ -12,6 +12,8 @@
 
 $(O_OBJ):	../raw/rawdataset.h
 
+../o/ceos.lo:	CFLAGS	+= -O1
+
 clean:
 	rm -f *.o $(O_OBJ)
 
